modules/retail/pages/retail-uyuni-server-setup.adoc

```diff
@@ -0,0 +1,291 @@
+[[retail-server-setup]]
+= Retail Uyuni Server Setup
+
+This section covers {productname} {smr} Server setup, using these procedures:
+
+* Start {productname} setup with {yast}
+* Create the main administration account with the {productname} {webui}
+* Name your base organization and add login credentials
+* Synchronize the openSUSE Leap channel
++
+// ^^^ CHECKIT
+
+
+
+[[retail-server-setup-yast]]
+== Set up {productname} with {yast}
+
+This section guides you through {productname} setup procedures.
+
+.Procedure: {productname} Setup
```

Comment:
I think this won't render properly. You'll need to do a local build to check.

Reply:
astonishingly, it builds fine :)  antora and pdf!